js/script.js

```diff
@@ -69,14 +69,15 @@
 
 
 // GDPR script
+/*
```

Comment:
Let's delete this section if it's not used anymore

Reply:
I just did!